RealmMapView/RealmMapView.swift

```diff
@@ -249,6 +259,27 @@ public class RealmMapView: MKMapView {
         rlmConfiguration.schemaVersion = configuration.schemaVersion
         return rlmConfiguration
     }
+  
+    public func config(entityName : String!, latitudeKeyPath : String!, longitudeKeyPath : String!, titleKeyPath : String!, subtitleKeyPath : String!) {
+        self.entityName = entityName
+        self.latitudeKeyPath = latitudeKeyPath
+        self.longitudeKeyPath = longitudeKeyPath
+        self.titleKeyPath = titleKeyPath
+        self.subtitleKeyPath = subtitleKeyPath
+    }
+  
+    private func assertConfig() -> Bool {
+        if self.entityName == nil
+            || self.latitudeKeyPath == nil
+            || self.longitudeKeyPath == nil
+            || self.titleKeyPath == nil
+            || self.subtitleKeyPath == nil {
+                print("You MUST configure the RealmMapView at the StoryBoard or use the config(...) method")
```

Comment:
I would have this read:

"Warning: You MUST configure RealmMapView properties: entityName, latitudeKeyPath, and longitudeKeyPath before refreshMapView is called. Aborting refresh."